trin-state/src/jsonrpc.rs

```diff
@@ -23,6 +23,24 @@ impl StateRequestHandler {
                     let radius = &self.network.overlay.data_radius;
                     let _ = request.resp.send(Ok(Value::String(radius.to_string())));
                 }
+                StateEndpoint::FindContent => {
```

Comment:
Well there definitely is plenty of duplicated logic here. In fact, all of the logic for the 3 currently supported endpoints seem to be identical. But, I'm not convinced the solution is to extract it into common methods. If we're doing that, then why are we splitting the logic out into separate modules in the first place? My intuition is that they are identical currently, but as the needs and nuances of the respective networks develop - then they'll require differentiation. So it seems premature to extract the data. But, this could also just be me justifying my laziness. What do you think?

Reply:
Maybe you are right, I'm happy to leave this as it is. We can address it when we start to add other networks in the stack :)